build.gradle

```diff
@@ -185,11 +176,11 @@ checkstyleTest.enabled(false)
     resolutionStrategy {
       eachDependency { DependencyResolveDetails details ->
         if (details.requested.group in ['com.fasterxml.jackson.core', 'com.fasterxml.jackson.module', 'com.fasterxml.jackson.datatype']) {
-          details.useVersion '2.10.1'
+          details.useVersion '2.12.0'
         }
 
         if (details.requested.name == 'jackson-databind') {
-          details.useVersion '2.10.2'
+          details.useVersion '2.12.0'
         }
```

Comment:
shouldn't be needed as you have the group exclusion

Reply:
Yep made the necessary changes.